indexing-hadoop/src/main/java/io/druid/indexer/path/HadoopGlobPathSplitter.java

```diff
@@ -31,9 +35,12 @@
    * e.g. splitGlob("/a,/b") -> ["/a","/b"]
    * splitGlob("/a/{c,d}") -> ["/a/c", "/a/d"]
    */
-  public static List<StringBuilder> splitGlob(String path)
+  public static Iterable<String> splitGlob(String path)
   {
-    return splitGlob(new CharStream(path));
+    if (path.indexOf('{') < 0) {
+      return Arrays.asList(path.split(","));
+    }
```

Comment:
yes, it would only be needed if normal flow was very inefficient in processing this case.